examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/main/MainScreen.kt

```diff
@@ -69,25 +70,35 @@ private fun MainNavHost(
     navigateToPaywallFooterScreen: (Offering?) -> Unit,
     navigateToPaywallCondensedFooterScreen: (Offering?) -> Unit,
     navigateToPaywallByPlacementScreen: (String) -> Unit,
-    modifier: Modifier = Modifier,
+    contentPadding: PaddingValues,
 ) {
     NavHost(
         navController,
         startDestination = Tab.Paywalls.route,
-        modifier = modifier,
     ) {
         composable(Tab.AppInfo.route) {
-            AppInfoScreen()
+            AppInfoScreen(
+                modifier = Modifier
+                    .padding(contentPadding)
+                    .consumeWindowInsets(contentPadding),
```

Comment:
Hmm I'm wondering, would it make more sense not pass the contentPadding as a parameter, but instead just use the [safeContentPadding](https://developer.android.com/reference/kotlin/androidx/compose/ui/Modifier#(androidx.compose.ui.Modifier).safeContentPadding()) modifier? 

That's supposed to respect unconsumed safe area and also handles consuming the safe area insets by itself.

Reply:
interestingly I tried that and it adds some unexpected padding 🤔 

<img width="257" height="158" alt="Screenshot 2025-08-01 at 15 41 31" src="https://github.com/user-attachments/assets/431985eb-4538-4d95-8cd0-0e965e7b58c9" />

Reply:
This works though:
```
modifier = Modifier.systemBarsPadding(),
```